server/coordinator/procedure/create_partition_table.go

```diff
@@ -20,30 +20,34 @@ import (
 // │ Begin  ├─────▶ CreatePartitionTable ├─────▶  CreateDataTables  ├─────▶OpenPartitionTables ├─────▶  Finish   │
 // └────────┘     └──────────────────────┘     └────────────────────┘     └────────────────────┘     └───────────┘
 const (
-	eventCreatePartitionTable = "EventCreatePartitionTable"
-	eventCreateDataTables     = "EventCreateDataTables"
-	eventOpenPartitionTables  = "EventOpenPartitionTables"
-	eventFinish               = "EventSuccess"
-
-	stateBegin                = "StateCreatePartitionTableBegin"
-	stateCreatePartitionTable = "StateCreatePartitionTable"
-	stateCreateDataTables     = "StateCreatePartitionTableCreateDataTables"
-	stateOpenPartitionTables  = "StateOpenPartitionTables"
-	stateFinish               = "StateCreatePartitionTableFinish"
+	eventCreatePartitionTable        = "EventCreatePartitionTable"
+	eventCreateDataTables            = "EventCreateDataTables"
+	eventOpenPartitionTablesMetaData = "EventOpenPartitionTablesMetaData"
```

Comment:
What does it mean by "open partition tables meta data"?

Reply:
It means change the mapping of table to shard in cluster metadata, which is implemented by calling the `OpenTable` function in cluster, so i named it `open partition tables meta data`. Now I rename it as `StateUpdateTableShardMetadata`